editor/editor.gd

```diff
@@ -442,7 +442,7 @@ func _collection_displayer_can_drop_data(at_position: Vector2, data) -> bool:
 		return false
 	
 	if ref_block == collection_displayer.root:
-		collection_displayer.drop_mode_flags = Tree.DROP_MODE_ON_ITEM
+		collection_displayer.drop_mode_flags = Tree.DROP_MODE_INBETWEEN | Tree.DROP_MODE_ON_ITEM
```

Comment:
This is for root block, are you sure you want in between for that block?

Reply:
Yes

Reply:
Just confirmed, no issue allowing this